Dockerfile

```diff
@@ -12,6 +12,8 @@ WORKDIR /app
 EXPOSE 8080
 ENV PORT=8080 MIX_ENV=prod
 
+COPY ./scripts/load_secrets_and_run.sh load_secrets_and_run.sh
+
```

Comment:
i think this COPY is not required. the [ADD](https://github.com/artsy/aprd/blob/c6abf0508d35bac1877ddaafff6afbdd618485af/Dockerfile#L8) takes care of copying `scripts` dir. this Dockerfile has only one build stage, so all the files added by that ADD are kept in the final image. MP was different in that it has 2 stages ([#1](https://github.com/artsy/metaphysics/blob/d0d6ab07696d58582624b05eebb8d7e525e61e46/Dockerfile#L4), [#2](https://github.com/artsy/metaphysics/blob/d0d6ab07696d58582624b05eebb8d7e525e61e46/Dockerfile#L36)), and stage #2 is the final image and it has only those files that are explicitly [copied](https://github.com/artsy/metaphysics/blob/d0d6ab07696d58582624b05eebb8d7e525e61e46/Dockerfile#L51) from stage #1, therefore, the `COPY` is required there.

Reply:
Oh, I didn't know that we don't need the `ADD` command. I've removed it form the Dockerfile.